backend/main.py

```
@@ -0,0 +1,39 @@
+    return {"status": "ok", "service": "botifesp"}
+
+
+@app.post("/chat", response_model=ChatResponse)
+def chat(body: ChatRequest):
+    """
+    Endpoint principal. Recebe a pergunta do usuário,
+    passa pelo grafo LangGraph e retorna a resposta gerada localmente.
+    """
+    try:
+        state = botifesp_graph.invoke({"query": body.query})
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)